### ISSUE TITLE
**Feature Request**: Support Markdown in Footer Text

### ISSUE THREAD

**Issue by saurabh-mish** [imfing/hextra#24]
A user may want to add a link to the license that's applicable for the site. In this case a modified copyright text in the footer may look like:

```
© 2023 FirstName LastName - [Some License]("https://www.somelicense.com/version")
```

+ The date can be rendered using Hugo template instead of hardcoding

+ FirstName and LastName can be taken from a field in `hugo.yaml`

+ Link for a specific version of a license can be provided

**Comment by saurabh-mish** [comment]
This fix doesn't seem to work:

I did:

```
hugo mod get -u
rm -rf resources/ public/
hugo server --buildDrafts --disableFastRender --ignoreCache
```

[This](https://codeberg.org/mishra/ferris.rs/src/branch/main/i18n/en.yaml) is the file I've modified.

**Comment by imfing** [comment]
@saurabh-mish the update command will only update when I publish a new release

I've just published v0.2.4 which should contain the new feature
https://github.com/imfing/hextra/releases/tag/v0.2.4

Note: if you want to use the latest main branch, please do: `hugo mod get -u github.com/some/repo@main`

**Comment by saurabh-mish** [comment]
Thanks !!
